__tests__/app.test.js

```diff
@@ -199,4 +199,38 @@ describe('GET /api/articles/:article_id and comment count' ,() => {
                 })
         
             })
-        })
\ No newline at end of file
+        })
+        describe('GET /api/articles/:article_id/comments' ,() => {
+            test('status:200, responds with all the comments that match the article_id', () => {   
+                const article_id=1;
+                return request(app)
+                    .get(`/api/articles/${article_id}/comments`)
+                    .expect(200)
+                    .then(({body})=>{
+                        console.log(body.comments, "hsb");
```

Comment:
We need to check the length of an array before we start looping through it - otherwise, if it's empty, forEach will still think it's done it's job, but it won't return false anywhere because there will be no objects to check against.